docs/Encyclopedia/pages/Arriba.md

```diff
@@ -0,0 +1,31 @@
+# Arriba
+
+## Description ##
+
+The Arriba workflow is part of the GDC mRNA quantification analysis pipeline used for detecting gene fusions in RNA-Seq data. 
+
+## Overview ##
+
+Arriba is one of the two pipelines the GDC uses to detect gene fusions. The Arriba gene fusion pipeline uses Arriba v1.1.0 to detect gene fusions from the RNA-Seq data of tumor samples.
```

Comment:
We might want to shy away from specific versions here, in case they change.

Reply:
changed